cal/cal/templates/home_logged_in.html

```diff
@@ -26,6 +26,13 @@
         }
       });
     });
+    function runFullSync() {
+      if ("{{request.user.profile.first_time_authed}}" == "True") {
+        "{{request.user.profile.resetFirstTimeAuth}}";
+        window.location = "/v1/sync?full_sync=true&sync_all=true";
+      };
+    };
+    window.onload = runFullSync;
```

Comment:
Since you don't always run full sync on page load, the name of this function might be confusing. Maybe something like "syncIfFirstTimeAuth" or something? I trust your judgment.

Edit: This might not be relevant anymore after you angularize this code and understand the implications of inc sync